tools/ping/src/lib.rs

```diff
@@ -394,7 +428,7 @@ async fn ping_via_node(
         head_height,
         time::Duration::seconds(recv_timeout_seconds.into()),
     )
-    .await
+        .await
```

Comment:
is that indent intentional? :)

Reply:
Expect `cargo fmt` to fix it, but `cargo fmt` has no opinion about this indent.